<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>qTwitter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTableView" name="statusTableView" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="homeToolButton" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="resources.qrc" >
         <normaloff>:/icons/icons/home.png</normaloff>:/icons/icons/home.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="settingsToolButton" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="resources.qrc" >
         <normaloff>:/icons/icons/configure.png</normaloff>:/icons/icons/configure.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="updateToolButton" >
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="resources.qrc" >
         <normaloff>:/icons/icons/reload.png</normaloff>:/icons/icons/reload.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="StatusEdit" name="statusEdit" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>32</height>
        </size>
       </property>
       <property name="text" >
        <string>What are you doing?</string>
       </property>
       <property name="maxLength" >
        <number>140</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="countdownLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection" >
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text" >
        <string>140</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="indent" >
        <number>-1</number>
       </property>
       <property name="openExternalLinks" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StatusEdit</class>
   <extends>QLineEdit</extends>
   <header>statusedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>statusTableView</tabstop>
  <tabstop>updateToolButton</tabstop>
  <tabstop>settingsToolButton</tabstop>
  <tabstop>homeToolButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections/>
</ui>
